mods/dungeon_loot/README.txt

```diff
@@ -0,0 +1,7 @@
+Minetest Game mod: dungeon_loot
+============================
+See top-level LICENSE.txt for license information.
```

Comment:
The top level file says:
```
License of source code
----------------------
Copyright (C) 2010-2012 celeron55, Perttu Ahola <celeron55@gmail.com>
See README.txt in each mod directory for information about other authors.
```
Each mod should have it's own license.txt for consistency, it is needed to mention you as the author.